packager/react-packager/src/Resolver/polyfills/Array.prototype.es6.js

```diff
@@ -5,54 +5,51 @@
  * @polyfill
```

Comment:
Files with `@polyfill` don't get wrapped in a closure scope (i.e. they aren't modules) in the internal Facebook packager... @martinbigio is it OK to make the changes below, in React Native's source?